app/api/utility.py

```diff
@@ -4,7 +4,7 @@
 
 #  Neurobagel nodes
 NEUROBAGEL_NODES = os.environ.get(
-    "NB_NODES", "https://api.neurobagel.org/query/"
+    "NB_NODES", "https://api.neurobagel.org/"
```

Comment:
Do we need to update some docs somewhere as well to make sure users don't accidentally add the `/query` back in their `.env` files?

Reply:
Yea good catch, I'll update the README.md.